index.js

```diff
@@ -122,3 +129,9 @@ module.exports = function requireDir(dir, opts) {
 
     return map;
 };
+
+function toCamelcase(str) {
+    return str.replace(/[_-][a-z]/ig, function (s) {
+        return s.substring(1).toUpperCase();
+      })
```

Comment:
:+1: , but minor code style: this last `})` line should be de-dented one level, and missing semicolon.